app/gen-server/migration/1756918816559-ServiceAccounts.ts

```diff
@@ -0,0 +1,97 @@
+import { MigrationInterface, QueryRunner, Table, TableColumn, TableForeignKey, TableIndex } from "typeorm";
+import * as sqlUtils from "app/gen-server/sqlUtils";
+
+export class ServiceAccounts1756918816559 implements MigrationInterface {
+
+  public async up(queryRunner: QueryRunner): Promise<any> {
+    const dbType = queryRunner.connection.driver.options.type;
+    const datetime = sqlUtils.datetime(dbType);
+
+    await queryRunner.addColumns('users', [
+      new TableColumn({
+        name: 'type',
+        type: 'varchar',
+        isNullable: false,
+        default: "'login'",
```

Comment:
The default is helpful for the migration but remains in effect afterwards which could be confusing?
I like the alternative approach taken in older migrations such as https://github.com/gristlabs/grist-core/blob/main/app/gen-server/migration/1753088213255-GroupTypes.ts#L21-L27 where the column is added as nullable, filled in and then set to non-nullable.